### PR TITLE
Support GHC-8.2

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

**Comment by phadej** [comment]
Ping @fizruk, are you ok with going with `cabal-doctest` for `doctests`?

**Comment by fizruk** [comment]
@phadej anything to make it work :)

**Comment by phadej** [comment]
@fizruk, could you recheck. I can then make the release.

**Comment by phadej** [comment]
ping @fizruk

File: swagger2.cabal [hunk 1]
```diff
@@ -93,11 +97,11 @@ test-suite spec
     Data.Swagger.Schema.ValidationSpec
   default-language: Haskell2010
 
-test-suite doctest
-  build-depends:    base, doctest, Glob
+test-suite doctests
+  build-depends:    base, doctest, Glob, QuickCheck
```
Comment (phadej) [inline swagger2.cabal]: we have one `prop>`, it requires `QuickCheck` then.

This is a little hack, actually we don't need `QuickCheck` to build `doctests.hs`; **but** this dependencies are solved by Cabal, so we can pass them as `--package-id=...` to `doctests` executable, so `doctest` can use / import `QuickCheck` modules.

Comment (fizruk) [inline swagger2.cabal]: Maybe add this comment to the file then? Or a link to this conversation?

Comment (phadej) [inline swagger2.cabal]: I'll add an entry to `cabal-doctest` FAQ/Notes and add a link here

Comment (phadej) [inline swagger2.cabal]: Made a PR. https://github.com/phadej/cabal-doctest/pull/16 and added a link to where it will land.

**Comment by phadej** [comment]
Released:  https://hackage.haskell.org/package/swagger2-2.1.4.1

**Pull request merged**